src/OSPSuite.Core/Events/WeightObservedDataChangedEvent.cs

```diff
@@ -0,0 +1,18 @@
+﻿using OSPSuite.Core.Domain;
+using OSPSuite.Core.Domain.Data;
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace OSPSuite.Core.Events
+{
+   public class WeightObservedDataChangedEvent
```

Comment:
WeightObservedDataChangedEvent => 
WeightedObservedDataChangedEvent

Match up the file name after.